app/controllers/settings_controller.rb

```diff
@@ -16,14 +16,17 @@ def index
     end
 
     resp = {}
+    @schools = School.active.map do |s| 
+      [s.name + " (#{s.code[1..-1].upcase})", s.id] 
+    end
 
-    unless current_user.nil?
+    if current_user.present?
       @email = current_user.email
       @alt_email = current_user.alt_email
       @contact_email = current_user.contact_email
       @school = current_user.school
       @holds = current_user.holds.order("created_at DESC")
-
+      @schools << ["[INACTIVE] #{@school.name} (#{@school.code[1..-1].upcase})", @school.id] unless @school.active
```

Comment:
this is plural?

Reply:
@daryanypl inactive school pushing to active schools. @schools is array